src/sausage-rolls.md

```diff
@@ -1,25 +1,33 @@
 # Sausage Rolls
 
+Times varies greatly depending on oven temperature, I recommend 150 Celcius for upwards of an hour to the effect that the inside meat is throughly cooked. 
```

Comment:
Add Fahrenheit as well. Consider mentioning this temperature again in the instructions.